package.json

```diff
@@ -6,11 +6,16 @@
     "type": "git",
     "url": "git://github.com/mozilla/normandy.git"
   },
+  "bin": {
+    "stylelint": "./node_modules/stylelint/dist/cli.js"
+  },
   "scripts": {
     "watch": "webpack --config ./webpack.config.js --watch",
     "build": "webpack --config ./webpack.config.js",
     "test": "NODE_ENV=production karma start",
-    "lint": "eslint normandy client webpack.config.js karma.conf.js"
+    "lint": "npm run lint-js; npm run lint-css;",
+    "lint-js": "eslint normandy client webpack.config.js karma.conf.js --fix || true",
+    "lint-css": "stylelint '**/*.s?(a|c)ss' --config .stylelintrc || true"
```

Comment:
I think instead of putting this here we should just handle all of this in therapist. You'd need to update the therapist.yml file to:
```yaml
actions:
  flake8:
    run: flake8 {files}
    include: "*.py"
    exclude: "docs/"
  eslint:
    run: ./node_modules/.bin/eslint {files} --fix
    include: 
      - "*.js"
      - "*.jsx"
  stylelint:
    run: ./node_modules/bin/stylelint {files}
    include:
      - "*.css"
      - "*.scss"
```